packages/application-components/src/components/custom-views/custom-views-selector/custom-views-selector.tsx

```diff
@@ -34,8 +41,68 @@ const Wrapper = styled.div<TWrapperProps>`
 `;
 
 const Container = styled.div`
-  background-color: ${designTokens.colorAccent98};
-  padding: ${designTokens.spacing25} ${designTokens.spacing60};
+  background: linear-gradient(
+      0deg,
+      ${designTokens.colorNeutral95},
+      ${designTokens.colorNeutral95}
+    ),
+    linear-gradient(
+      0deg,
+      ${designTokens.colorNeutral98},
+      ${designTokens.colorNeutral98}
+    );
+  padding: 7px ${designTokens.spacing20};
```

Comment:
Nit: why `7`? Can't we use the normal scale of multiple of 4s (4, 8, 12, 16, ...)?

Reply:
Sounds good to me 👍, would it be ok with you @NataliaSclipet?
<img width="1324" alt="Screenshot 2023-11-28 at 10 06 50" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/75ecee60-16e2-4765-b00f-91ae0f43e707">